x-pack/plugins/cases/server/services/user_actions/audit_logger.ts

```diff
@@ -31,7 +32,7 @@ export class UserActionAuditLogger {
       event: {
         action: event.descriptiveAction,
         category: ['database'],
-        type: [actionsToEcsType[event.action]],
+        type: [actionsToEcsType[event.action] as string],
```

Comment:
I don't think this casting is required?
```suggestion
        type: [actionsToEcsType[event.action]],
```